lib/view_model/wallet_list/wallet_edit_view_model.dart

```diff
@@ -45,11 +46,11 @@ abstract class WalletEditViewModelBase with Store {
     final walletService = getIt.get<WalletService>(param1: wallet.type);
     await walletService.remove(wallet.name);
     resetState();
-    _walletListViewModel.updateList();
   }
 
   @action
   void resetState() {
```

Comment:
make it private and call it from rename instead of directly calling `updateList`